pandas_schema/schema.py

```diff
@@ -29,12 +29,14 @@ def __init__(self, columns: typing.Iterable[Column], ordered: bool = False):
         self.columns = list(columns)
         self.ordered = ordered
 
-    def validate(self, df: pd.DataFrame, columns: typing.List[str] = None) -> typing.List[ValidationWarning]:
+    def validate(self, df: pd.DataFrame, columns: typing.List[str] = None, allow_sub_schema: bool = False) -> \
```

Comment:
Also please split this over multiple lines so it doesn't need a linebreak escape